server/model/config.go

```diff
@@ -1309,20 +1310,36 @@ func (s *LogSettings) SetDefaults() {
 		s.FileJson = NewBool(true)
 	}
 
+	if s.AdvancedLoggingJSON == nil {
+		s.AdvancedLoggingJSON = []byte("{}")
+	}
+
 	if s.AdvancedLoggingConfig == nil {
 		s.AdvancedLoggingConfig = NewString("")
 	}
 }
 
+func (s *LogSettings) GetAdvancedLoggingConfig() []byte {
+
+	if len(s.AdvancedLoggingJSON) != 0 && string(s.AdvancedLoggingJSON) != "{}" {
+		return s.AdvancedLoggingJSON
+	}
+	if s.AdvancedLoggingConfig != nil && *s.AdvancedLoggingConfig != "" {
+		return []byte(*s.AdvancedLoggingConfig)
+	}
+	return []byte("{}")
```

Comment:
If we are always returning "{}" from here, is it still needed to check for `\"\"` inside `IsEmptyDSN`?

Reply:
I've created a IsEmptyJSON utility that checks for any valid JSON document that is empty (i.e.  [], "", {})